lib/logger/lib/logger.ex

```diff
@@ -871,14 +871,14 @@ defmodule Logger do
   def __should_log__(level, module) do
     level = Logger.Handler.elixir_level_to_erlang_level(level)
 
-    if process_allowed?(level, self()) and :logger.allow(level, module) do
+    if :logger.allow(level, module) do
       level
     end
   end
 
-  defp process_allowed?(level, pid) do
+  @doc false
```

Comment:
Let's inline this function inside `Logger.Filter.process_level` and we are good to go!